pages/_app.tsx

```diff
@@ -4,7 +4,6 @@ import Head from 'next/head';
 import { useRouter } from 'next/router';
 import { MDXProvider } from '@mdx-js/react';
 import { Footer } from '../components/Footer';
-import { Header } from '../components/Header';
```

Comment:
This component doesn't exist in the master branch?

Reply:
I deleted it, it's not needed anymore.